tests/test_new_features.py

```
@@ -0,0 +1,514 @@
+        import alcove.query.api as api_mod
+        importlib.reload(api_mod)
+
+        client = TestClient(api_mod.app)
+        r = client.get("/view", params={"source": "sample.txt"})
+        assert r.status_code == 200
+
+        # The /view call should have been logged with doc_bytes
+        # Make another request to force flush
+        client.get("/health")
+        if log_file.exists():
+            entries = [json.loads(l) for l in log_file.read_text().strip().splitlines()]
```

Ruff (0.15.5)

[error] Ambiguous variable name: `l`

(E741)